### ISSUE TITLE
[Feature Request] Notification Integration with Discord

### ISSUE THREAD

**Issue by ahylick** [prysmaticlabs/prysm-web-ui#132]
Are there any plans to integrate with discord for notifications?

**Comment by prestonvanloon** [comment]
Short answer: Not at the moment.

Long answer: Could you please elaborate on the type of notifications or types of messages you would like? I think that integrating with discord is very easy to do and would make for a great first contributor issue. A well scoped request will make it easier for this feature to be implemented. 

TL;DR - What would you like to see on discord?

**Comment by ahylick** [comment]
TL;DR - I want to message a private Discord server each time my validators propose a block.

complete context:
I'm a solo staker with 1 Windows server validator node and 1 Ubuntu server validator node. For my linux validator, I have Grafana setup with an alert configured to message a private Discord server upon receiving a block proposal opportunity. Since I don't have Grafana for my windows server, I'm trying to use prysm-web as a substitute. However, I don't have the native alert-to-discord pathway that grafana afforded me on my linux machine. Thus, I requested this functionality from prysm-web.

This alert notification to discord is simply a cheap SMS notification substitute for letting me know when I've gotten block rewards from my validators.

Let me know if I need to explain the usecase more.

**Comment by MareG76** [comment]
It should be possible to choose where we want to receive alerts Discord, Telegram or email? Notificaton about proposed block, missed block, validator service down...I know there is similar function in Grafana but it would be nice to have it on prysm-web-ui

**Comment by sharedStake-dev** [comment]
+1 . SharedStake will offer a small bounty for this.
